lib/schemas/swaref.xsd

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?> 

```

Comment:
The reference to this schema is from src/com/sun/ts/tests/jaxws/common/xml/catalog/META-INF/jax-ws-catalog.xml + bin/com/sun/ts/tests/jaxws/common/xml/catalog/META-INF/jax-ws-catalog.xml.

Do we have a link for the WS-I license terms?